### ISSUE TITLE
File field: Embedding a file will result to Trouble processing request. Try again after refreshing the page.

### ISSUE THREAD

**Issue by JbalTero** [directus/api#1238]
## Steps to Reproduce

* Development env
* Create `pdfs` collection with `downloadable` file field
* Add pdfs item and use these links below to embed in the `downloadable` field
    * https://cdn.fileinfo.com/img/ss/lg/jpg_44.jpg
    * http://www.africau.edu/images/default/sample.pdf

## Expected Behavior

They should be embedded successfully

## Actual Behavior

`Trouble processing request. Try again after refreshing the page.`

```
Notice: Trying to get property of non-object in /src/core/Directus/Filesystem/Files.php on line 284
```

## Other Context & Screenshots

[screenrecording](https://drive.google.com/file/d/1assMeLDLxWnuqk-cbABKhfj6jdQ_dW7L/view?usp=sharing)

## Technical Details

- Device: Desktop
- OS: Win 10  10.0.18362.145
- Web Server: Apache/2.4.29
- PHP Version: PHP 7.1.30
- Database: 10.2.26
- Install Method: v2.5.0 build
- Directus App: 7.10.0

**Comment by bjgajjar** [comment]
@JbalTero 

Currently, we have the support of Vimeo and Youtube as embedded URL support. You can find more info from [here](https://docs.directus.io/api/reference.html#files).

As per the current support, If the provided URL is not a link from Vimeo or Youtube; it will be considered as an Image. PDF support is not available in the current system. 

This should be considered as an enhancement instead of a bug,

**Comment by JbalTero** [comment]
tahnks

**Comment by bjgajjar** [comment]
Fixed in #1245

**Comment by JbalTero** [comment]
thanks
